pkg/controller/containersource/resources/deployment.go

```diff
@@ -47,7 +47,6 @@ func MakeDeployment(org *appsv1.Deployment, args *ContainerArguments) *appsv1.De
 			Namespace:    args.Namespace,
 		},
 		Spec: appsv1.DeploymentSpec{
-			Replicas: func() *int32 { var i int32 = 1; return &i }(),
```

Comment:
Was this intended to be removed?

If not I think the following is clearer.

```go
Replicas: int32(1)
```

Reply:
It was intended to be removed.

`Spec.Replicas` is a pointer to an int to distinguish between unset and the valid value `0`. Go doesn't allow pointers to int literals so this function _thing_ was necessary to create an int pointer in one line.

I removed it because:
- It eliminates that inline function
- we always set `Replicas` to 1 which is the default for ReplicaSet anyway
- I think the correct behavior is to inherit the default number of replicas, whatever that happens to be (and it might not always be 1).